### ISSUE TITLE
Close button for TagItem control doesn't have "name" attribute

### ISSUE THREAD

**Issue by KarynaLisniak12** [microsoft/fluentui#15317]
### Environment Information

- **Package version(s)**: 7.93.0

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/karynalisniak12/pen/bGpymgy

#### Actual behavior:

"name" attribute is not assigned to “Close” button of TagItem element.

#### Expected behavior:

"name" attribute should be assigned to “Close” button of TagItem element. 
Users will not be able to know the detail of the Remove button if they want to remove tag.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

**Comment by paulgildea** [comment]
@KarynaLisniak12 Thanks for the issue.

@JustSlone will follow up with the solution here.

**Comment by JustSlone** [comment]
@KarynaLisniak12 actually if you just set the `removeButtonAriaLabel` prop on the `TagPicker` component it controls the aria label for the remove button.
I've updated your codepen with the change: https://codepen.io/justslone-the-encoder/pen/yLOWwjp?editors=0010

```jsx
        <TagPicker 
          onRenderItem={props => <TagItem {...props}><span>{props.item.name}</span></TagItem>}
          onResolveSuggestions={this._onFilterChanged}
          getTextFromItem={item => item.name}
          removeButtonAriaLabel="Remove" // This controls the AriaLabel for the remove button
          itemLimit={2}
        />
```

**Comment by KarynaLisniak12** [comment]
@JustSlone All right, but we need in "name" attribute. Accessibility error detection tools consider the absence of the "name" attribute to be an error. So, we need to add an attribute to the "Close" button and fix this problem.

**Comment by JustSlone** [comment]
@KarynaLisniak12 can you point to the spec/requirements for the "name" attribute? Maybe I'm missing something, but it seems like the requirement is for aria-label, not name. see: [ARIA spec 1.2 on name calculation](https://www.w3.org/TR/wai-aria-1.2/#namecalculation)

**Comment by KarynaLisniak12** [comment]
We discussed this problem with team members and decided that "name" attribute isn't necessary for us. Thanks for your help!

**Comment by JustSlone** [comment]
Great, glad you're unblocked. I'm going to close this issue out, if there is more here feel free to re-open or file a new issue

Thanks!
